Jamfile

```diff
@@ -511,10 +553,20 @@ feature.compose <encryption>off : <define>TORRENT_DISABLE_ENCRYPTION ;
 feature mutable-torrents : on off : composite propagated link-incompatible ;
 feature.compose <mutable-torrents>off : <define>TORRENT_DISABLE_MUTABLE_TORRENTS ;
 
-feature crypto : built-in openssl libcrypto gcrypt : composite propagated ;
+feature crypto : built-in openssl libcrypto gcrypt wolfssl : composite propagated ;
 feature.compose <crypto>openssl : <define>TORRENT_USE_LIBCRYPTO <define>TORRENT_USE_OPENSSL <define>OPENSSL_NO_SSL2 ;
 feature.compose <crypto>libcrypto : <define>TORRENT_USE_LIBCRYPTO ;
 feature.compose <crypto>gcrypt : <define>TORRENT_USE_LIBGCRYPT ;
+feature.compose <crypto>wolfssl
+	: <define>TORRENT_USE_WOLFSSL
+	<define>TORRENT_USE_LIBCRYPTO
```

Comment:
I don't think we would need to rely on libcrypto emulation, right? does anything fail if you don't define this?

Reply:
well, is either this or incorporate your commit :), and the main benefit of using wolfSSL is using it all (for SSL and Crypto)

Reply:
and for what I see there is no "libcrypto emulation", just "extra" and "all" openssl emulation, which both are needed for ASIO.

Reply:
I see. but it emulates libcrypto API in the sense that the SHA-1 and SHA-256 etc. algorithm can be used as if we had libcrypty, right? because you're not linking against libcrypto in your branch

Reply:
true